testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -4108,6 +4108,19 @@ static void checkRlikeFails(SqlOperatorFixture f) {
     f.checkNull("CHARACTER_LENGTH(cast(null as varchar(1)))");
   }
 
+  @Test void testLenFunc() {
```

Comment:
Could use a comment - that LEN is a snowflake-specific alias for CHAR_LENGTH, and that snowflake also supports LENGTH

Reply:
Added